datasets/lv/sanction_list/lv_sanction_list.yml

```
@@ -0,0 +1,28 @@
+name: lv_sanction_list
+title: Latvia List of Sanctions
```

```suggestion
name: lv_fiu_sanctions
title: Latvia FIU Sanctions
```

could you update the file and directory names too?